common/app/common/commercial/PrebidIndexSite.scala

```diff
@@ -262,11 +268,24 @@ object PrebidIndexSite {
     ).groupBy { case (section, _) => section }
       .mapValues { _.map { case (_, site) => site }.toSet }
 
-  private def fromSectionId(sectionId: String): Option[Set[PrebidIndexSite]] = siteIds.get(sectionId)
+  private def fromSectionId(sectionId: String): Option[Set[PrebidIndexSite]] = {
+    val firstPart = {
+      val sepIndex = sectionId.indexOf('/')
+      if (sepIndex > 0) sectionId.substring(0, sepIndex) else sectionId
+    }
+    siteIds.get(firstPart).orElse(Some(defaultSiteIds))
```

Comment:
If it always going to be a `Some`, perhaps make this function return a `Set[PrebidIndexSite]` and wrap the result of this in the Some in other places you need it as that type?

Reply:
I think it has to be optional to avoid breaking the press, as this gets stored as part of the pressed page.